{{ cookiecutter.service_name }}/internal/api/handlers/handler/routes.go

```diff
@@ -0,0 +1,7 @@
+package handler
+
+import "github.com/labstack/echo/v4"
+
+func (h Handler) RegisterRoutes(e *echo.Group) {
+
```

Comment:
Add a comment with an example on to generate routes here

v1 := e.Group("/v1") 
v1.GET("/stuff", h.getStuff)